frontend/src/console.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { BrowserRouter } from "react-router-dom"
```

Comment:
Why do you need react router if you can just use and modify `currentUrl` of `userContext`?

Reply:
I thought that's the way to do it/ how would you do it differently?

Reply:
See https://github.com/RazEfron/top_ten/pull/2#discussion_r471020723